src/t_string.c

```diff
@@ -228,14 +228,14 @@ int parseExtendedStringArgumentsOrReply(client *c, int *flags, int *unit, robj *
         } else if (!strcasecmp(opt,"PERSIST") && (command_type == COMMAND_GET) &&
                !(*flags & OBJ_EX) && !(*flags & OBJ_EXAT) &&
                !(*flags & OBJ_PX) && !(*flags & OBJ_PXAT) &&
-               !(*flags & OBJ_KEEPTTL))
+               !(*flags & OBJ_KEEPTTL) && !(*flags & OBJ_PERSIST))
```

Comment:
how about this (extra line without touching the line), but its not critical
```suggestion
               !(*flags & OBJ_PERSIST) &&
               !(*flags & OBJ_KEEPTTL))
```